tests/lib/util/glob-util.js

```diff
@@ -29,7 +30,7 @@ var fixtureDir;
  */
 function getFixturePath() {
     var args = Array.prototype.slice.call(arguments);
-    args.unshift(fixtureDir);
```

Comment:
This code is also in https://github.com/eslint/eslint/blob/master/tests/lib/cli-engine.js#L53-L57.  Should this same change be made there?